ocaml/src/nsm_model.ml

```diff
@@ -1598,7 +1579,9 @@ module NamespaceManager(C : Constants)(KV : Read_key_value_store) = struct
       Update'.compare_and_swap
         (Keys.objects ~object_id)
         None
-        (Some (serialize Manifest.output manifest)) in
+        (Some (serialize (Manifest.to_buffer
+                            ~version:1 (* TODO: should become 2 *)) manifest))
```

Comment:
"TODO: should become 2" ... once all nsm nodes have upgraded that is? I guess the TODO can stay then for now.